files/default/neutron-ha-tool.py

```diff
@@ -415,10 +422,10 @@ def l3_agent_migrate(qclient, noop=False, now=False):
 
     total_errors = 0
     total_migrations = 0
-    for agent_id in agent_dead_list:
+    for agent in agent_dead_list:
         (migrations, errors) = \
             migrate_l3_routers_from_agent(
-                qclient, agent_id, agent_alive_list, noop)
+                qclient, agent, agent_alive_lqist, noop, wait_for_ports)
```

Comment:
You have a typo here, that will break things `s/agent_alive_lqist/agent_alive_list`